ssz/sedes/serializable.py

```diff
@@ -0,0 +1,419 @@
+import abc
+import collections
+import copy
+import re
+
+from eth_utils import (
+    to_dict,
+    to_set,
+    to_tuple,
+)
+
+from ssz.constants import (
+    CONTAINER_PREFIX_LENGTH,
+)
+from ssz.exceptions import (
+    DeserializationError,
+    SerializationError,
+)
+
+
+class MetaBase:
+    fields = None
+    field_names = None
+    field_attrs = None
+
+
+def _get_duplicates(values):
+    counts = collections.Counter(values)
+    return tuple(
+        item
+        for item, num in counts.items()
+        if num > 1
+    )
+
+
+def validate_args_and_kwargs(args, kwargs, arg_names, allow_missing=False):
```

Comment:
```suggestion
def validate_args_and_kwargs(args, kwargs, arg_names):
```
`allow_missing` (here as well as in `merge_kwargs_to_args`) doesn't seem to be used